index.bs

```diff
@@ -643,13 +643,12 @@ The <dfn method for=CookieStore>get(|options|)</dfn> method steps are:
 1. Let |origin| be the [=current settings object=]'s [=/origin=].
 1. If |origin| is an [=opaque origin=], then return [=a promise rejected with=] a "{{SecurityError}}" {{DOMException}}.
 1. Let |url| be the [=current settings object=]'s [=creation URL=].
-1. If |options|' {{CookieStoreGetOptions/url}} dictionary member is present, then run these steps:
-    1. Let |parsed| be the result of [=basic URL parser|parsing=] |options|' {{CookieStoreGetOptions/url}} dictionary member with [=/this=]'s [=relevant settings object=]'s [=API base URL=].
-    1. If the [=current global object=] is a {{Window}} object and |parsed| does not [=url/equal=] |url|,
-        then return [=a promise rejected with=] a {{TypeError}}.
-    1. If |parsed|'s [=url/origin=] and |url|'s [=url/origin=] are not the [=same origin=],
-        then return [=a promise rejected with=] a {{TypeError}}.
-    1. Set |url| to |parsed|.
+1. Let |parsed| be the result of [=basic URL parser|parsing=] |options|' {{CookieStoreGetOptions/url}} dictionary member with [=/this=]'s [=relevant settings object=]'s [=API base URL=].
```

Comment:
Even though the _options_ argument is now required, the `url` member is still optional, so this algorithm shouldn't change, should it?

Reply:
you are right 🤦  change reverted.